cg/meta/demultiplex/housekeeper_storage_functions.py

```diff
@@ -238,3 +231,13 @@ def get_sample_sheets_from_latest_version(flow_cell_id: str, hk_api: Housekeeper
     except HousekeeperBundleVersionMissingError:
         sample_sheet_files: List = []
     return sample_sheet_files
+
+
+def get_sample_sheet_path_hk(flow_cell_id: str, hk_api: HousekeeperAPI) -> Path:
+    """Returns the path to the sample sheet for the given bundle."""
+    sample_sheet_files: List[File] = get_sample_sheets_from_latest_version(
```

Comment:
```suggestion
    sample_sheet_files: List[File] = get_sample_sheets_from_latest_version(
```
Why would this be a list? I think we do not allow multiple sample sheets for a flow cell. or is the function a more generic one. Likely out of scope but good to think about.

Reply:
True, I did not write the function so I'm not sure. Something we can refactor

Reply:
another PR sounds good.